lib/net/http/header.rb

```diff
@@ -145,23 +324,46 @@ def each_header   #:yield: +key+, +value+
 
   alias each each_header
 
-  # Iterates through the header names in the header, passing
-  # each header name to the code block.
+  # Calls the block with each field key;
+  # returns the value of #to_hash;
+  # see {Fields}[rdoc-ref:Net::HTTPHeader@Fields]:
+  #
+  #   req = Net::HTTP::Get.new(uri)
+  #   req.each_key {|key| p key }
+  #
+  # Output:
+  #
+  #   "accept-encoding"
+  #   "accept"
+  #   "user-agent"
+  #   "host"
   #
   # Returns an enumerator if no block is given.
+  #
+  # Net::HTTPHeader#each_name is an alias for Net::HTTPHeader#each_key.
```

Comment:
I don't think we need to use the full module name when we're in that module.

```suggestion
  # #each_name is an alias for #each_key.
```

(Also technically, it's the other way around: `#each_key` is an alias for `#each_name`).

Reply:
I'm aware, but am trying to use one term -- key -- as much as possible, to avoid confusion between key and name.

Reply:
> I don't think we need to use the full module name when we're in that module.

The doc guide is explicit on this point.  So, stet.